src/main/java/edu/usf/sas/pal/muser/model/MusicAnalysisModel.java

```diff
@@ -30,16 +30,201 @@ public MusicAnalysisModel() {
     public static class AlbumArtistData {
 
         public AlbumArtistData() {
+
+        }
+
+        public static class AlbumData {
+
+            @Override
+            public String toString() {
+                return "AlbumData [albumArtistName=" + albumArtistName + ", artists=" + artists + ", dateAdded="
+                        + dateAdded + ", id=" + id + ", name=" + name + ", numDiscs=" + numDiscs + ", numSongs="
+                        + numSongs + ", paths=" + paths + ", year=" + year + "]";
+            }
+
+
+            public AlbumData() {
+            }
+
+
+            public String getAlbumArtistName() {
+                return albumArtistName;
+            }
+
+            public void setAlbumArtistName(String albumArtistName) {
+                this.albumArtistName = albumArtistName;
+            }
+
+            private String albumArtistName;
+            private List<Artists> artists;
+            private Long dateAdded;
+            private Long id;
+            private String name;
+            private Integer numDiscs;
+            private Integer numSongs;
+            private List<String> paths;
+            private Integer year;
+
+            public static class Artists {
+
+                @Override
+                public String toString() {
+                    return "id=" + id + ", name=" + name + ", numAlbums=" + numAlbums + ", numSongs="
+                            + numSongs + "";
+                }
+
+
+                public long id;
+                public String name;
+                public int numAlbums;
+                public int numSongs;
+
+                public long getId() {
+                    return id;
+                }
+
+                public String getName() {
+                    return name;
+                }
+
+                public int getNumAlbums() {
+                    return numAlbums;
+                }
+
+                public int getNumSongs() {
+                    return numSongs;
+                }
+
+                public void setId(long id) {
+                    this.id = id;
+                }
+
+                public void setName(String name) {
+                    this.name = name;
+                }
+
+                public void setNumAlbums(int numAlbums) {
+                    this.numAlbums = numAlbums;
+                }
+
+                public void setNumSongs(int numSongs) {
+                    this.numSongs = numSongs;
+                }
+
+
+                public Artists() {
```

Comment:
Constructors are generally declared as the first function in the class. Please move `Artists()` before `toString` function.